dashboard/src/pages/TreeDetails/Tabs/Boots/BootsTab.tsx

```diff
@@ -24,6 +23,8 @@ import MemoizedIssuesList from '@/components/Cards/IssuesList';
 import MemoizedHardwareTested from '@/components/Cards/HardwareTested';
 import type { TestsTableFilter } from '@/types/tree/TreeDetails';
 
+import TreeCommitNavigationGraph from '../TreeCommitNavigationGraph';
```

Comment:
relative import, we need the linter rule

Reply:
I thought we would use relative import when getting from the parent directory, I'll change it